kogito-editors-js/docs/new-component.md

```diff
@@ -0,0 +1,105 @@
+# How to create a Kogito editor React component
+
+All Kogito editor React components live in the `packages` directory and follow the convention of **components** and **loaders**.
+
+- **Components** - all elements an editor may be a marshaller, a dialog, a Boxed Expression editor, or even a FEEL editor.
+
+- **Loaders** - this is more specific. It teaches how each editor (DMN or BPMN) may load their components.
+
+In this guide, we will create a **component** and add it to GWT in 5 steps.
+
+### 1. Create a new component
+
+Let's use the following script to create the minimal boilerplate code that a React component needs:
+
+```bash
+yarn new-component
+```
+
+Once you execute this, insert the name of your new component. In my case, In my case, I will call it `feel-dialog`.
+
+### 2. Run the development web app
+
+As I named my component as `feel-dialog`, I type this command to access it:
+
+```bash
+cd packages/feel-dialog-component
+```
+
+> **Notice**: The directory already has the proper suffix by convention!
+
+Now you may run this to install the default dependencies of your component and development web app:
+
+```bash
+yarn install
+
+cd showcase
+yarn install
```

Comment:
maybe we should use `yarn bootstrap` as we use in `kogito-tooling`? or here is the `install` autogenerated and we can not change it?

Reply:
We're at a sub-package level in this context, and we're relying on one of Yarn's default commands (`install`) to proceed.

But, we do have the `yarn bootstrap` script that relies on Lerna (just like Kogito Tooling) at the root level :-)